plugins/algoltek-usb/README.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin: Algoltek USB
+---
+
+## Introduction
+
+This plugin supports the firmware upgrade of DisplayPort over USB-C to HDMI converter provided by Algoltek, Inc. These DisplayPort over USB-C to HDMI converters can be updated through multiple interfaces, but this plugin is only designed for the USB interface.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a packed binary file format.
+
+This plugin supports the following protocol ID:
+
+* `tw.com.algoltek.usb`
+
+## GUID Generation
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+* `USB\VID_25A4&PID_9421`
+
+## Update Behavior
+
+The firmware is deployed when the device is in normal runtime mode, and the device will reset when the new firmware has been programmed.
+
+## Vendor ID Security
+
+The vendor ID is set from the USB vendor, in this instance set to `USB:0x25A4`.
+
+## External Interface Access
+
+This plugin requires read/write access to `/dev/bus/usb`.
+
+## Version Considerations
+
+This plugin has been available since fwupd version `2.0.0`.
```

Comment:
Maybe we should make this 1.9.11 -- I think you probably want this backported to the stable branch -- 2.0.0 isn't planned for a couple of months and you probably want this in the hands of users before that.

Reply:
Definitely yes. We hope this plugin can be added for us to move on next stage, the WWCB.